packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -94,6 +96,9 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 			...properties
```

Comment:
I think this should be moved after the next line of code (`this.isFooterEnabled = xxxx`).

That way this can be adapted like this:

```typescript
this._columnProperties = (input || []).map((properties: StarkTableColumnProperties) => ({
    ...DEFAULT_COLUMN_PROPERTIES,
    ...properties,
    footerValue: this.isFooterEnabled ? (properties.footerColumn || '') : undefined
}));
```

So in case the footer is enabled, the footerValue of the current column is set to whatever the devevloper passed to it or '' so that it is displayed in the column

Reply:
mhhh I did try, but since then I haven't been able to get the footer working anymore.

Reply:
In the code I wrote above I have a typo: `properties.footerColumn` but it should be `properties.footerValue` :p

Reply:
Hi @christophercr,
I think I know why it's not working, if I put the code above under 
`		this.isFooterEnabled =
			findIndex(
				this.columnProperties,
				(column: StarkTableColumnProperties) => typeof column.footerValue !== "undefined" && column.footerValue !== ""
			) > -1;`
this.`columnProperties`, requiers - >`this._columnProperties` set later in the code line 117
`	public get columnProperties(): StarkTableColumnProperties[] {
		return this._columnProperties;
	}`

Maybe I'm wrong ? Any idea how i can solve this ?

Reply:
Oh right... well then I think it is still possible to move it above but the `isFooterEnabled` should be adapted like this:

```typescript
		this.isFooterEnabled =
			findIndex(
				(input || []),
				(column: StarkTableColumnProperties) => typeof column.footerValue !== "undefined" && column.footerValue !== ""
			) > -1;
```

Basically just replace `this.columnProperties` by `(input || [])` which is the same we do in the code above

Tip: you could replace also `findIndex()` by the array [some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) function 😉